### ISSUE TITLE
License header not correctly templated

### ISSUE THREAD

**Issue by poikilotherm** [asdf-vm/asdf-plugin-template#19]
**Describe the bug**

When using `setup.bash`, choosing an MIT license does not replace `[year]` and `[full name]` in the license text.

**Steps to reproduce**

1. Create a new repo from this template.
2. Use `setup.bash` and choose MIT as your license.
3. Take a look at the generated LICENSE file.

**Expected behavior**

```diff
-Copyright (c) [year] [fullname]
+Copyright (c) 2020 Oliver Bertuch
```

**Comment by jthegedus** [comment]
I do not think this is a viable soln as some licenses have require more user input than this, eg: `agpl-3.0` has this placeholder also:

><one line to give the program's name and a brief idea of what it does.>

I will PR a soln that adds a TODO item into the license instead.

**Comment by jthegedus** [comment]
closed in #38
